pkg/src/message.js

```diff
@@ -56,6 +56,12 @@ export function printMessage(m, pkg) {
     case 'EXPORTS_TYPES_SHOULD_BE_FIRST':
       // prettier-ignore
       return `${c.bold(fp(m.path))} should be the first in the object as required by TypeScript.`
+    case 'EXPORTS_MODULE_SHOULD_PRECEED_REQUIRE':
+      // prettier-ignore
+      return `${c.bold(fp(m.path))} should come before 'require' so it can take precedence when used by a bundler.`
+    case 'EXPORTS_MODULE_SHOULD_PRECEED_IMPORT':
+      // prettier-ignore
+      return `${c.bold(fp(m.path))} should come before 'import' so it can take precedence when used by a bundler.`
```

Comment:
With the above combined, we can merge this message too so they'll get one errors instead of two.

The message also needs to be added to the website code at https://github.com/bluwy/publint/blob/master/site/src/utils/message.js